membrane_curvature/lib/mods.py

```diff
@@ -1,150 +1,44 @@
 import itertools as it
 import numpy as np
-import pickle
-import math
 
 
-def def_all_beads(lipid_types, leaflets, head_list, topology):
-    """
-    Select reference elements to derive membrane surface.
-
+def grid_map(coords, factor):
+    """ Maps coordinates to grid.
 
     Parameters
     ----------
-    lipid_types : list.
-        List of lipid types in simulation box.
-    leaflets : str. Default ["lower", "upper"]
-        Leaflets of bilayer.
-    head_list : list
-        List of indexes by leaflet. Provided by user in -ii and -io.
-    topology: obj
-        Topology from grofile.
-    Returns
-    -------
-    [ element_1, ..., element_n] : list
-        List with elements used as a reference to derive membrane surface.
+    x: float
+        Value of x coordinate
+    y: float
+        Value of y coordinate
```

Comment:
Docs (x,y) do not seem to agree with function signature (coords, factor)

Reply:
Yes, you are right and thanks for this catch. I'll update it.